runtime/tests/jniargsystemlinkage/module.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+   Copyright (c) 2019, 2019 IBM Corp. and others
+
+   This program and the accompanying materials are made available under
+   the terms of the Eclipse Public License 2.0 which accompanies this
+   distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+   or the Apache License, Version 2.0 which accompanies this distribution and
+   is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+   This Source Code may also be made available under the following
+   Secondary Licenses when the conditions for such availability set
+   forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+   General Public License, version 2 with the GNU Classpath
+   Exception [1] and GNU General Public License, version 2 with the
+   OpenJDK Assembly Exception [2].
+
+   [1] https://www.gnu.org/software/classpath/license.html
+   [2] http://openjdk.java.net/legal/assembly-exception.html
+
+   SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<module xmlns:xi="http://www.w3.org/2001/XInclude">
+  <xi:include href="../jniarg/exports.xml">
+    <xi:fallback/>
```

Comment:
Please remove `<xi:fallback/>`: if the include fails, we *want* an error.

Reply:
Fixed in aa851c4 (see [force push](https://github.com/eclipse/openj9/compare/5f938ef88bb3f8d33239c8a3c05657370baa457e..ab713cac6f91859325f0e62136936b0bb7ddbec8)).